accounts-db/src/accounts_db.rs

```diff
@@ -1091,6 +1091,10 @@ pub struct AccountStorageEntry {
     approx_store_count: AtomicUsize,
 
     alive_bytes: AtomicUsize,
+
+    /// offsets to accounts that are zero lamport single ref stored in this storage.
+    /// These are still alive. But, shrink will be able to remove them.
+    zero_lamport_single_ref_offsets: RwLock<IntSet<usize>>,
```

Comment:
Do we need to keep all the offsets here?
Is the count of the number of single-ref zeros in the storage enough?